query.go

```diff
@@ -8,6 +8,48 @@ import (
 	"time"
 )
 
+// NullString represents a string that may be null.
+type NullString struct {
+	String string
+	Valid  bool // Valid is true if String is not NULL
```

Comment:
Where are these `Valid` bools actually checked? I could find them uses in test code. Is the flag meant to be checked by the client of this library?

Reply:
> Is the flag meant to be checked by the client of this library?

Yes, the client if this library can check the `Valid` field to know if the value is nil or not.

Reply:
Let me actually include an example in the README, I forgot to do that.

Reply:
Here's how it can be used when using the `database/sql` package: https://pkg.go.dev/database/sql#NullString

It can be used the same way.